pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -570,6 +580,7 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				authorizationapi.NewRule(readWrite...).Groups(imageGroup).Resources("imagestreamimages", "imagestreammappings", "imagestreams", "imagestreams/secrets", "imagestreamtags").RuleOrDie(),
 				authorizationapi.NewRule("create").Groups(imageGroup).Resources("imagestreamimports").RuleOrDie(),
 				authorizationapi.NewRule("get", "update").Groups(imageGroup).Resources("imagestreams/layers").RuleOrDie(),
+				authorizationapi.NewRule("update").Groups(imageGroup).Resources("images/signatures").RuleOrDie(),
```

Comment:
images are cluster scoped, which means that images/signatures is also cluster scoped, which means that a RegistryAdmin (project scoped) won't be able to do this.